languages/sk.js

```diff
@@ -0,0 +1,79 @@
+ */
+(function () {
+	'use strict';
+
+	// Replace <code> with the language code, e.g. no, fr, en, ect.
+	sceditor.locale['sk'] = {
+
+		// Original string is on the left, place the translation between
+		// the quotes on the right
+		'Bold': 'Tučné',
+		'Italic': 'Kurzíva',
+		'Underline': 'Podčiarknuté',
+		'Strikethrough': 'Prečiarknuté',
+		'Subscript': 'Dolný index',
+		'Superscript': 'Horný index',
+		'Align left': 'Zarovnať vľavo',
+		'Center': 'Zarovnať na stred',
+		'Align right': 'Zarovnať vpravo',
+		'Justify': 'Zarovnať do bloku',
+		'Font Name': 'Typ písma',
+		'Font Size': 'Veľkosť písma',
+		'Font Color': 'Farba písma',
+		'Remove Formatting': 'Odstrániť formátovanie',
+		'Cut': 'Vystrihnúť',
+		'Your browser does not allow the cut command. Please use the keyboard shortcut Ctrl/Cmd-X': 'Váš prehliadač nepodporuje príkaz pre vystrihnutie. Prosím, použite klávesovú skratku Ctrl/Cmd-X',
+		'Copy': '',
+		'Your browser does not allow the copy command. Please use the keyboard shortcut Ctrl/Cmd-C': 'Váš prehliadač nepodporuje príkaz pre skopírovanie. Prosím, použite klávesovú skratku Ctrl/Cmd-C',
+		'Paste': '',
+		'Your browser does not allow the paste command. Please use the keyboard shortcut Ctrl/Cmd-V': 'Váš prehliadač nepodporuje príkaz pre vloženie. Prosím, použite klávesovú skratku Ctrl/Cmd-V',
+		'Paste your text inside the following box:': 'Vložte Váš text do nasledujúceho poľa',
+		'Paste Text': 'Vložiť text',
+		'Bullet list': 'Zoznam',
+		'Numbered list': 'Číslovaný zoznam',
+		'Undo': 'Krok späť',
+		'Redo': 'Krok vpred',
+		'Rows:': 'Riadkov:',
+		'Cols:': 'Stĺpcov:',
+		'Insert a table': 'Vložiť tabuľku',
+		'Insert a horizontal rule': 'Vložiť vodorovnú čiaru',
+		'Code': 'Vložiť kód',
+		'Width (optional):': 'Šírka (voliteľné):',
+		'Height (optional):': 'Výška (voliteľné):',
+		'Insert an image': 'Vložiť obrázok',
+		'E-mail:': 'E-mail:',
+		'Insert an email': 'Vložiť E-mail',
+		'URL:': 'URL adresa:',
+		'Insert a link': 'Vložiť odkaz',
+		'Unlink': 'Zrušiť odkaz',
+		'More': 'Viac',
+		'Insert an emoticon': 'Vložiť smajlíka',
+		'Video URL:': 'URL adresa videa:',
+		'Insert': 'Vložiť',
+		'Insert a YouTube video': 'Vložiť YouTube video',
+		'Insert current date': 'Vložiť dnešný dátum',
+		'Insert current time': 'Vložiť aktuálny čas',
+		'Print': 'Vytlačiť',
+		'View source': 'Zobraziť zdrojový kód',
+		'Description (optional):': 'Popis (voliteľné):',
+		'Enter the image URL:': 'Vložte URL adresu obrázka:',
+		'Enter the e-mail address:': 'Vložte E-mailovú adresu:',
+		'Enter the displayed text:': 'Vložte zobrazovaný text:',
+		'Enter URL:': 'Vložte URL adresu:',
+		'Enter the YouTube video URL or ID:': 'Vložte URL adresu YouTube videa alebo jeho ID:',
+		'Insert a Quote': 'Vložiť citát',
+		'Invalid YouTube video': 'Neplatné YouTube video',
+		'Left-to-Right': 'Zľava doprava',
+		'Right-to-Left': 'Zprava doľava',
+		'Drop files here': 'Presuňte súbory sem',
+		
```

Comment:
There's some extra white space here that's causing the linter to complain, could you remove it? Also if you could add an empty line to the end as it complains if there isn't one.